pkg/volume/csi/csi_mounter.go

```diff
@@ -218,8 +202,13 @@ func (c *csiMountMgr) SetUpAt(dir string, mounterArgs volume.MounterArgs) error
 	}
 
 	// create target_dir before call to NodePublish
-	if err := os.MkdirAll(dir, 0750); err != nil && !corruptedDir {
-		return errors.New(log("mounter.SetUpAt failed to create dir %#v:  %v", dir, err))
+	if err := os.MkdirAll(dir, 0750); err != nil {
```

Comment:
We should Mkdir the parent(dir) to align with the csi spec wording. I believe then we don't need the corruptedDir check because the parent dir is out of the driver's control